pkg/axfr/axfr.go

```diff
@@ -59,32 +64,31 @@ func (s *Lookup) DoAXFR(name, server string) AXFRServerResult {
 		s.Factory.Factory.BlMu.Lock()
 		if blacklisted, err := s.Factory.Factory.Blacklist.IsBlacklisted(server); err != nil {
 			s.Factory.Factory.BlMu.Unlock()
-			retv.Status = "ERROR"
+			retv.Status = string(zdns.STATUS_ERROR)
 			retv.Error = "blacklist-error"
 			return retv
 		} else if blacklisted {
 			s.Factory.Factory.BlMu.Unlock()
-			retv.Status = "ERROR"
+			retv.Status = string(zdns.STATUS_ERROR)
```

Comment:
Curious if there's a specific reason to keep the status and other fields in the `AXFRServerResult` as strings, not some sort of typed value.

Reply:
Updated to use `zdns.Status`